package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "url-shortener",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "devStart": "nodemon server.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "ejs": "^3.0.1",
+    "express": "^4.17.1",
+    "mogoose": "5.9.1",
```

Typo. Replace `mogoose` with `mongoose`


```suggestion
    "mongoose": "5.9.1",
```